lib/src/io.dart

```diff
@@ -1240,8 +1239,12 @@ class PubProcessResult {
 
   // TODO(rnystrom): Remove this and change to returning one string.
```

Comment:
Is this TODO really resolved?

Reply:
No this one isn't, so I left it here. It would require a larger refactoring.

Reply:
Ah sorry, this PR is about the other TODO. Gotcha....